exercises/practice/pythagorean-triplet/.meta/example.rb

```diff
@@ -1,52 +1,19 @@
-class Triplets
-  attr_reader :factors, :sum
-  def initialize(conditions)
-    min = conditions.fetch(:min_factor) { 1 }
-    max = conditions.fetch(:max_factor)
-    @sum = conditions[:sum]
-    @factors = (min..max).to_a
-  end
-
-  def to_a
+class PythagoreanTriplet
+  def self.triplets_with_sum(sum)
     triplets = []
-    each_triplet do |triplet|
-      triplets << triplet if select?(triplet)
+    (2..(sum / 2)).to_a.reverse.each do |c|
```

Comment:
Funny, or interesting, that Rubocop considers the space around `/` here, but does not consider "too many parenthesis".  I wrote (and then deleted) a mention about this line with the "Perfect use of no space around `/`" because it communicates well the idea of "half of sum" as a fraction.

But `(2..sum/2).to_a` is valid, the lack of space around `/` communicating the right thing in terms of its "closeness" to `sum`.

Anyway, thought I would mention it here, since the change was made (and merged).

Reply:
Yeah, I did that intentionally (same with the `a*a` lower down), but I'm totally fine with just going with whatever Rubocop wants here :)